app/Http/Controllers/RecommendationController.php

```diff
@@ -9,26 +9,34 @@
 use App\Models\User;
 use App\Models\Email;
 
-
 class RecommendationController extends \Controller
 {
     protected $settings;
 
     protected $rules = [
-    'rank_character'  => 'required',
-    'rank_additional' => 'required',
-    'essay1'          => 'required',
+      'rank_character'  => 'required',
+      'rank_additional' => 'required',
+      'essay1'          => 'required',
+    ];
+
+    protected $applicant_rules = [
```

Comment:
hmm, I little confused by this, it seems like you're only ever checking for `applicant_rules` but isn't `rules` required when a rec fills out the form?
should this be like a `applicant_rules` and a `recommender_rules` array, but also if a recommender can now edit their own fields, those values would also be required. so both need to be true for a recommender to submit?